x-pack/solutions/observability/plugins/observability/common/constants.ts

```diff
@@ -39,3 +39,7 @@ export const OBSERVABILITY_RULE_TYPE_IDS_WITH_SUPPORTED_STACK_RULE_TYPES = [
   ...OBSERVABILITY_RULE_TYPE_IDS,
   ...STACK_RULE_TYPE_IDS_SUPPORTED_BY_OBSERVABILITY,
 ];
+
+export enum ALERTS_API_URLS {
```

Comment:
Since you created this enum, I am wondering if you could replace the usage of `internal/observability/alerts/related_dashboards` with the newly created `ALERTS_API_URLS.INTERNAL_RELATED_DASHBOARDS`

- https://github.com/elastic/kibana/blob/main/x-pack/solutions/observability/plugins/observability/server/routes/alerts/route.ts#L21
- x-pack/test/api_integration/deployment_agnostic/apis/observability/incident_management/suggested_dashboards.ts

This is definitely not a blocker and can come as a follow up PR

Reply:
I see you had already replaced the alerts api url in the route!